client/app/scripts/components/node-details/node-details-health-item.js

```diff
@@ -10,8 +10,8 @@ class NodeDetailsHealthItem extends React.Component {
       <div className="node-details-health-item">
       <div className="node-details-health-item-value">{formatMetric(this.props.value, this.props)}</div>
         <div className="node-details-health-item-sparkline">
-          <Sparkline data={this.props.samples} max={this.props.max}
-            first={this.props.first} last={this.props.last} />
+          <Sparkline data={this.props.wire_metrics.samples} max={this.props.wire_metrics.max}
+            first={this.props.wire_metrics.first} last={this.props.wire_metrics.last} />
```

Comment:
How do you suggest we do this? I guess I can create a custom serializer which obtains the values from the underlying type. Let me give it a try.